R/plotLoadings.R

```diff
@@ -0,0 +1,385 @@
+#' Plot feature loadings after performing a reducedDim 
```

Comment:
Change this to: "Plot feature loadings for TreeSummarizedExperiment objects."

Then later in @details explain that the TreeSE object is expected to have content in reducedDim slot.

However TreeSE inherits SingleCellExperiment, and SingleCellExperiments have reducedDim slot (but no tree). Could we make this more generic and add support for SCE objects?